recipes/r-concatenate/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = '1.0.0' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-concatenate
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/concatenate_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/concatenate/concatenate_{{ version }}.tar.gz
+  sha256: 1ca273eeb25a29fffe25b389cdbf22474b556010dfe4e3dc49160ec5f7e17acf
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('concatenate')"           # [not win]
+    - "\"%R%\" -e \"library('concatenate')\""  # [win]
+
+about:
+  home: https://github.com/jamesdunham/concatenate
+  license: GPL-3.0-or-later
+  summary: Simple functions for joining strings. Construct human-friendly messages whose elements
+    aren't known in advance, like in stop, warning, or message, from clean code.
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+
+# Package: concatenate
+# Title: Human-Friendly Text from Unknown Strings
+# Version: 1.0.0
+# Authors@R: person("James", "Dunham", email = "james@jamesdunham.io", role = c("aut", "cre"))
+# Description: Simple functions for joining strings. Construct human-friendly messages whose elements aren't known in advance, like in stop, warning, or message, from clean code.
+# Depends: R (>= 3.1.0)
+# Imports: methods
+# License: GPL (>= 3.2)
```

Comment:
This is problematic. We have to get clarification on this first. https://github.com/jamesdunham/concatenate/issues/2

Reply:
Given the age of that package and lack of activity on the GitHub page, I'm going to drop `r-concatenate` from this PR so I can get the other packages merged.

Reply:
@mfansler I see that I added `r-concatenate` because `r-gwasrapidd` needed it.  Should we drop both packages in that case?